lms/static/sass/views/_login-register.scss

```diff
@@ -350,6 +350,8 @@ $sm-btn-linkedin: #0077b5;
         @extend %btn-secondary-grey-outline;
         width: 130px;
         padding: 0 0 0 ($baseline*2);
+        margin-right: 6px;
```

Comment:
I believe the UX team is trying to get away from specific pixel values like this in favor of predefined variables that handle spacing. I'm not sure if it will be requested here or not, since nothing else in the file is so updated, so maybe they'd rather handle it in a future task. Just a heads up. I'm sure @talbs or someone else on that team may clarify if it's needed.